src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -17,12 +17,14 @@ export const toolLabels: Record<ToolLabel, ToolLabel> = {
   spark: 'spark',
   JupyterLab: 'JupyterLab',
   Galaxy: 'Galaxy',
-  Cromwell: 'Cromwell'
+  Cromwell: 'Cromwell',
+  CromwellOnAzure: 'CromwellOnAzure'
 }
 
 export const appToolLabelTypes: Record<AppToolLabel, AppToolLabel> = {
   Galaxy: 'Galaxy',
-  Cromwell: 'Cromwell'
+  Cromwell: 'Cromwell',
+  CromwellOnAzure: 'CromwellOnAzure'
```

Comment:
I would think it would be better to display a nicer label to users. Probably 'Cromwell' is enough.

Reply:
I believe the mockups say something like **Workflows** on Cromwell (to give us an easier transition to a non-Cromwell-only world.

Reply:
I had a question about this. Is it possible to just change what is displayed in UI to `Workflows on Cromwell` but in code call it `CromellOnAzure`? I tried changing the right hand side of this line to `Workflows on Cromwell` but then it shows error because it doesn't recognize it. Maybe I am missing something simple 🤔

Reply:
Yeah, it does seem like this should be the place to supply the user-visible IA. Hopefully IA folks can chime in and explain.

Reply:
Something like this should work:

```
export type AppToolLabel = 'Galaxy' | 'Cromwell' | 'CromwellOnAzure'
export type ToolLabelDisplay = 'Workflows on Cromwell'
...

export const toolLabels: Record<ToolLabel, ToolLabel> = {
  ...
  CromwellOnAzure: 'CromwellOnAzure'
}

export const toolLabelDisplays: Partial<Record<ToolLabel, ToolLabelDisplay>> = {
  CromwellOnAzure: 'Workflows on Cromwell'
}
```

Good catch. Up until now the toolLabels have also been the display value.

Reply:
I see. And can you please point to me how I can use this new `toolLabelDisplays`? Simply replacing it in where it requires `toolLabel` might not work 🤔

Reply:
```
export type ToolLabelDisplay = ToolLabel |'Workflows on Cromwell'

export const toolLabels: Record<ToolLabel, ToolLabel> = {
  Jupyter: 'Jupyter',
  RStudio: 'RStudio',
  terminal: 'terminal',
  spark: 'spark',
  JupyterLab: 'JupyterLab',
  Galaxy: 'Galaxy',
  Cromwell: 'Cromwell',
  CromwellOnAzure: 'CromwellOnAzure'
}

export const toolLabelDisplays: Record<ToolLabel, ToolLabelDisplay> = {
  Jupyter: 'Jupyter',
  RStudio: 'RStudio',
  terminal: 'terminal',
  spark: 'spark',
  JupyterLab: 'JupyterLab',
  Galaxy: 'Galaxy',
  Cromwell: 'Cromwell',
  CromwellOnAzure: 'Workflows on Cromwell'
}
```

Then in ContextBar.js line 98:
div({ style: { fontWeight: 'bold' } }, `${toolLabelDisplays[toolLabel]} Environment`),

Reply:
Updated code to reflect this